web/src/app/workplace/workplace.module.ts

```diff
@@ -4,7 +4,7 @@ import { FormsModule } from '@angular/forms';
 import { HTTP_INTERCEPTORS, HttpClientModule } from '@angular/common/http';
 import { AngularSvgIconModule } from 'angular-svg-icon';
 import { WorkplaceRoutingModule } from './workplace-routing.module';
-import { AlertModule, TypeaheadModule } from 'ngx-bootstrap';
+import { AlertModule, TypeaheadModule, BsDatepickerModule } from 'ngx-bootstrap';
```

Comment:
move this module to shared since it's a cross functionality module.

Reply:
done!